pytest-embedded-serial-esp/pytest_embedded_serial_esp/serial.py

```diff
@@ -202,16 +169,23 @@ def wrapper(self, *args, **kwargs):
     def _start(self):
         self.hard_reset()
 
-    @use_esptool(hard_reset_after=True, no_stub=True)
     def hard_reset(self):
         """Hard reset your espressif device"""
-        pass
+        self.esp.hard_reset()
 
     @use_esptool()
     def erase_flash(self):
         """Erase the complete flash"""
         logging.info('Erasing the flash')
-        self.stub.erase_flash()
+        esptool.main(['erase_flash', '--force'], esp=self.esp)
```

Comment:
@hfudev Should we use `self.esp_flash_force` here too?

Reply:
yes, I agree.